packaging/DISTRIBUTIONS.md

```diff
@@ -0,0 +1,38 @@
+# Netdata distribution support matrix
```

Comment:
Since you added a new file, you'll need to update `docs/generator/buildyaml.sh` as well. Otherwise, the link won't work.
See https://deploy-preview-6422--netdata.netlify.com/DISTRIBUTIONS/, it's a 404.

Reply:
wow, i didn't know about that, checking